tests/test_flows/test_flow_functions.py

```diff
@@ -334,6 +335,7 @@ def test_get_flow_reinstantiate_model_wrong_version(self):
             assert "0.19.1" not in flow.dependencies
 
     def test_get_flow_id(self):
+        openml.utils._list_all = functools.lru_cache()(openml.utils._list_all)
```

Comment:
Do you think this could be parametrized with slow/fast as discussed offline this morning?

Reply:
Yes, will try to add that. Sorry for the confusion in opening the PR. I needed it as a non-draft PR because I was interested in the CI results.